### PR TITLE
Volvo pr Ciel day 5

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

File: script/Arcueid/Day 6/06_00_ARC06_3_2.txt [hunk 1]
```diff
@@ -85,7 +85,7 @@ The one responsible for that is, of course, Akiha.
 -- 単なる被害妄想、否、俺が気を遣いすぎているだけなのだが、それはそれとして社会的な生命の危機を感じるので、東館の二階には入らなかった。
 // I think 被害妄想 might be closer to persecution complex, just because it
 // seems more ingrained (被害) than the other definitions for paranoia (偏執病, 妄想症)?
-Maybe I've developed a persecution complex― no, I'm sure I was just overthinking it.
+Maybe I've developed a persecution complex―― no, I'm sure I was just overthinking it.
```
Comment (Avuxo) [inline script/Arcueid/Day 6/06_00_ARC06_3_2.txt:88]: this isn't an em-dash, it's signifying that his thought was cut-off

File: script/Ciel/Day 5/05_01_CIEL05_1B.txt [hunk 2]
```diff
@@ -1,25 +1,25 @@
 [sha:f19385a8030557e0d809dea7d66506c0591e7f3b]{
 -- Page 10002, Offset 11019. Mods: @k, @e.
 -- 「……なんだそりゃ。
-"What a load of bullshit...
+"What a load of bullshit...%{n}
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1B.txt:4]: why newline'd?

File: script/Ciel/Day 5/05_01_CIEL05_1B.txt [hunk 3]
```diff
@@ -52,6 +52,7 @@ Alright then, you want a part-time job?
 [sha:ca917979117674940e9e79c143f01a128cb28beb]{
 -- Page 63, Offset 11028.
 -- 「ヒュー！　勇敢な挑戦者一名様、ごあんなーい！」
+// can yeehaw be changed to yeah or something?
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1B.txt:55]: 🤠 don't destroy my culture, pardner

ヒュー could be "Fuck yeah!" I guess

File: script/Ciel/Day 5/05_01_CIEL05_1B.txt [hunk 4]
```diff
@@ -226,7 +227,7 @@ The man who kicked me――Haru――is looking down at me with a grin on his fa
 -- Page 75, Offset 11060.
 -- とにかく手足を動かして、岸を目指して泳ぐ。
 Ignoring him for the moment,
- I start kicking my arms and legs and swim toward the shore.
+ I start kicking my arms and legs, swimming toward the shore.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1B.txt:230]: revert

File: script/Ciel/Day 5/05_01_CIEL05_1C.txt [hunk 5]
```diff
@@ -186,6 +186,7 @@ I slam my head against the iron bars in anger.
 [sha:61775d325b1cd96cf258c64d9765ad04db73b3c1]{
 -- Page 100, Offset 11140.
 -- 人として正常な判断を、過剰な痛みで封殺する。
+// force those thoughts? is this supposed to be force out or drive out?
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1C.txt:189]: its separated by em dashes. The sentence is "I force those thoughts from my mind". No change necessary

File: script/Ciel/Day 5/05_01_CIEL05_1C.txt [hunk 6]
```diff
@@ -561,7 +562,7 @@ I put all of my strength into my petrified feet, and―――
 [sha:d78d1b822003d7e9ee00c4f4354c8668769cce39]{
 -- Page 121, Offset 11209.
 -- 決意と反応。どちらが先だったかは分からない。
-Was it a conscious decision, or instinct? I'm not sure which acted quicker.
+Was it a conscious decision or instinct? I'm not sure which acted quicker.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1C.txt:565]: I like the pause. revert

File: script/Ciel/Day 5/05_01_CIEL05_1C.txt [hunk 7]
```diff
@@ -596,7 +597,7 @@ Glancing around, I get a handle on the space I'm in.
 [sha:92c11d62ea2b6c72f2bc7d262ff5942b274a79aa]{
 -- Page 123, Offset 11216.
 -- 大きさにして直径６メートル、高さ２メートル弱の大部屋。
-It's a large circular room, roughly six meters in diameter, and a just shy of two meters tall.
+It's a large circular room, roughly six meters in diameter, and just shy of two meters tall.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1C.txt:600]: large, circular room

File: script/Ciel/Day 5/05_01_CIEL05_1D.txt [hunk 8]
```diff
@@ -103,7 +103,7 @@ Before I can even finish my question, she pins me up against the wall.
 [sha:ec0c07cfcdd7ab59d60b66ccdc426014a1085a36]{
 -- Page 157, Offset 11300.
 -- <ど|・><う|・><し|・><て|・>、<貴|・><方|・><は|・><そ|・><ん|・><な|・><に|・><平|・><気|・><な|・><の|・><よ|・>……！」
-Why are %{i}you%{/i} so calm...?!"
+%{i}Why are you so calm%{/i}...?!"
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1D.txt:106]: revert

File: script/Ciel/Day 5/05_01_CIEL05_1D.txt [hunk 9]
```diff
@@ -115,6 +115,7 @@ She beats her clenched fist against my chest with a thud.
 [sha:24c60793a788d34277af056e1d443b17b66e8ed3]{
 -- Page 158, Offset 11302.
 -- ノエル先生は半狂乱になりながら、涙目で、そんな言葉を口にした。
+// half-crazed or half-mad which do you prefer? it's half-crazed in the string summary so ill change it to be consistent
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1D.txt:118]: half-crazed is more american

File: script/Ciel/Day 5/05_01_CIEL05_1E.txt [hunk 10]
```diff
@@ -26,7 +26,7 @@ The ceiling is supported by numerous pillars.
 -- 天井中心部には窓らしき穴があり、灰色の陽差しを下ろしている。それこそ、雲の隙間からのぞく<天使の梯子|ジェイコブズラダー>のように。
 // Getting ~biblical~ are we
 Grey sunlight leaks in through a window-like hole in the center of the ceiling.
- It almost like a Jacob's Ladder peeking through the gaps in the clouds.
+ It's almost like Jacob's Ladder peeking through the gaps in the clouds.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1E.txt:29]: Note so sure about this. "Jacob's ladder", as in the ladder to heaven, isn't something you'd refer to as "peeking from between the clouds". I'd imagine this is referring to the light phenomenon which is why the old version was phrased the way it is.
image

File: script/Ciel/Day 5/05_01_CIEL05_1E.txt [hunk 11]
```diff
@@ -638,12 +638,12 @@ That thing's made of solid steel...
 [sha:8384e28fa2bdc188c02987c43e8d2e8cf22cc79f]{
 -- Page 248, Offset 11513. Mods: @k, @e.
 -- 「いいじゃない、細かいコトは♡
-"Relax, don't sweat the small stuff ♡
+"Relax, don't sweat the small stuff ♡%{n}
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_1E.txt:641]: again, can you explain why this one is newline'd?

File: script/Ciel/Day 5/05_01_CIEL05_2_2.txt [hunk 12]
```diff
@@ -491,7 +491,7 @@ To a man of the countryside, poverty is a virtue, isn't it?"
 [sha:87133cedcb369ca8adf4f817c4b66149ae53cfba]{
 -- Page 334, Offset 11757.
 -- 「“―――この国に踏み入った理由は、治療と、復讐だ。
-"―――The reasons I set foot in this country are... treatment, and revenge.
+"―――The reasons I set foot in this country are... treatment and revenge.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_2_2.txt:494]: pause was intentional. vlov speaks slow af. you could use an ellipses instead i suppose.

File: script/Ciel/Day 5/05_01_CIEL05_2_2B.txt [hunk 13]
```diff
@@ -12,7 +12,7 @@ At its core is―――
 [sha:d2722da1fd6935c1a85241d9b6c1eb5f83ec059d]{
 -- Page 10012, Offset 11825.
 -- 何か、
-something.
+%{s}something.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_2_2B.txt:15]: what is this one?

File: script/Ciel/Day 5/05_01_CIEL05_4.txt [hunk 14]
```diff
@@ -867,6 +867,7 @@ He also acquired ten wives to console him.
 [sha:97a15cd067de6c0fa4e8b1c839f6c0695b654312]{
 -- Page 724, Offset 12810.
 -- 主君への忠誠も、決して溶けぬ氷壁と同義だった。
+// is this fine to leave as lord, not sire or liege?
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4.txt:870]: i think it should probably be liege but i don't really remember

File: script/Ciel/Day 5/05_01_CIEL05_4B.txt [hunk 15]
```diff
@@ -134,7 +134,7 @@ Ice spears pour down like raindrops.
 [sha:8da6f46c8786e937b3e283c9a77ee6dac26ab712]{
 -- Page 740, Offset 12862.
 -- 槍はすべて俺に向けられている。ヴローヴにとって、俺はもう<そ|・><う|・><い|・><う|・><モ|・><ノ|・>として格上げされた。
-The spears are all aimed at me now. Vlov has identified me as the greater threat.
+The spears are all aimed at me now. Vlov has identified me as the %{i}greater%{/i} threat.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4B.txt:137]: no need for italics

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 16]
```diff
@@ -140,13 +140,13 @@ I mean, even with a gag order, there's going to be leaks, right?"
 "So long as we have a cover story, the rest is just a matter of legwork.
  Why d'ya think my old man's been leaning on people left and right?
  All you gotta do is put pressure on the guys at the top.
- Doesn't make a fat bit of difference if the small fry or middle managers get suspicious.
+ Doesn't make a fat bit of difference if the small fries or middle managers get suspicious.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:143]: we talked about this during ross's PR but yusuke and ross prefer "small fry" and it seems to be a valid plural form so revert.

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 17]
```diff
@@ -179,21 +179,21 @@ I mean, even with a gag order, there's going to be leaks, right?"
...
+"I haven't the slightest idea. Andou's private life holds no interest to me.
 }
 [sha:4d5e1406f183d7f3cc5b7de59cefb742bdfb0098]{
 -- Page 799, Offset 13015.
 -- ３年前、悪徳司祭の内部告発を決意したものの、最後の最後で金に目が眩んだ恋人に裏切られ、故郷を追われ、失意の果てに父方の祖国であるこの国に島流しになった、という事ぐらいしか。
 All I know is that three years ago,
  he decided to blow the whistle on a corrupt presbyter,
- but was betrayed at the final hour by his lover, who had been blinded by greed,
- after which he was driven from his hometown before,
+ but was betrayed at the final hour by his lover, who had been blinded by greed.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:195]: ross had wanted to keep this one a horrible run on sentence on purpose.

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 18]
```diff
@@ -398,7 +398,7 @@ Then why have they come to this island nation all on their own...?
 [sha:c1d027f9c58f36a2d664674436646a6356ced878]{
 -- Page 812, Offset 13051.
 -- 「アン？　なんだ、免許証でも見せてほしいのか？　テメェ、オレがガキだからって舐めてんのか？」
-"%{i}Huuuh%{/i}?
+"Huuuh?
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:401]: revert

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 19]
```diff
@@ -415,7 +415,7 @@ I-It's just... Umm, you seem... well... you know... too young to be the Cardinal
 [sha:43363c4ec437aca7817ae62d2899e9443c768d5f]{
 -- Page 814, Offset 13054.
 -- 「そりゃあ若ぇだろ。ジジイが百歳超えてから愛人に産ませたガキだからな。今年で12だ、おそれいったか。しかもまだ弟がいるって話だ。
-"Goddamn right I'm young.
+"Goddamn right, I'm young.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:418]: revert

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 20]
```diff
@@ -469,21 +469,21 @@ When you say 'opening act', which part was that exactly... sir?"
...
- You can always trust a third rate piece of shit to give their life top priority, can't ya?
+ You can always trust a third-rate piece of shit to give their life top priority, can't ya?
 }
 [sha:f58f44656ea28b85efab34432cd17d18ca153143]{
 -- Page 818, Offset 13064.
 -- 逃げ足だけは見所あるよ、オマエ。10年近く使い捨ての狗ころやってるヤツは筋金入ってやがる。オレが法王庁に帰り次第、テメェは首だ。修道院送りじゃねえぞ。ギロチン送りの方な」
 // 首を切る meaning to fire someone and to behead someone.
 Only thing you're any goddamn good at is running away.
- You'd think that after being a disposable attack dog for ten years you'd have grown a fucking backbone.
+ You'd think that after being a disposable attack dog for ten years, you'd have grown a fucking backbone.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:479]: revert

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Ciel/Day 5/05_01_CIEL05_4.txt [hunk 21]
```diff
@@ -43,7 +43,7 @@ Thank goodness, she's still alive...!
 [sha:851e5dabff809d236113e08efb422ff2a2180960]{
 -- Page 679, Offset 12660.
 -- 「聞こえますか!?　苦しいだろうけど起きて、シエル先輩！」
-"Can you hear me?! I know it must hurt, but you have to get up, Ciel-senpai!"
+"Can you hear me?! I know it must have hurt, but you have to get up, Ciel-senpai!"
```
Comment (rschlaikjer) [inline script/Ciel/Day 5/05_01_CIEL05_4.txt:46]: It's more like "I know you are in pain" than "I know that what happened to you hurt", so I'd revert

File: script/Ciel/Day 5/05_01_CIEL05_4.txt [hunk 22]
```diff
@@ -321,14 +321,14 @@ A reward? All I can think about is what's in front of me right now.
...
- I can't help it―――what kind of healthy young man %{i}wouldn't%{/i} get a little hot under the collar when presented with such a voluptuous body.
+ I can't help it―――what kind of healthy young man %{i}wouldn't%{/i} get a little hot under the collar when presented with such a voluptuous body?
 }
 [sha:d867cac7ee25be99dd5609c7e7fe9bb5c4345a44]{
 -- Page 696, Offset 12711.
 -- この体をどうしたいとか、そんなコトならいくらでも思い浮かぶ。でもきっと台無し、何もかも台無しだ。そもそもワンパンでおしまいだ。
 // Does 'one pump chump' seriously transcend language barriers lmao
 // Also this page is TIGHT
-What would I like to do with body?
+What would I like to do with that body?
```
Comment (rschlaikjer) [inline script/Ciel/Day 5/05_01_CIEL05_4.txt:331]:
```diff
- It'd ruin everything.
+ I'd ruin everything.
```
to make it fit

File: script/Ciel/Day 5/05_01_CIEL05_4.txt [hunk 23]
```diff
@@ -976,7 +977,7 @@ The link my consciousness had formed is severed, and I return to my senses.
 -- Page 730, Offset 12830.
 -- 「“おれの、何を視た…………！！！！！！！！！！！”」
 // Eleven !
-"What of mine did you see......!!!!!!!!!!!"
+"What of mine did you see......?!!!!!!!!!!!"
```
Comment (rschlaikjer) [inline script/Ciel/Day 5/05_01_CIEL05_4.txt:980]: Not all questions have to have ? and I will DIE on this hill

File: script/Ciel/Day 5/05_01_CIEL05_4B.txt (hunk 15 above)
Comment (rschlaikjer) [reply script/Ciel/Day 5/05_01_CIEL05_4B.txt:137]: or shift to "identified _me_ as the greater" if we feel like we have to

File: script/Ciel/Day 5/05_01_CIEL05_4B.txt [hunk 24]
```diff
@@ -83,7 +83,7 @@ At this rate, my brain will shut down from this headache long before I breathe m
 [sha:435fb04c2f386e6ba567ea32204d46faf253a9c6]{
 -- Page 737, Offset 12852.
 -- 「そうですか。では、<ま|・><だ|・><信|・><じ|・><て|・>いいんですね？」
-"I see. Then, I can still count on you, can't I?"
+"I see. Then, I can still %{i}count%{/i} on you, can't I?"
```
Comment (rschlaikjer) [inline script/Ciel/Day 5/05_01_CIEL05_4B.txt:86]: I couldn't think of any word to italicize here that doesn't just read weird in EN. Strictly it'd be "I can _still count_ on you can't I" but it's more confusing than useful so I'd just leave it dropped.

File: script/Ciel/Day 5/05_01_CIEL05_4C.txt [hunk 25]
```diff
@@ -74,7 +74,7 @@ I don't want to worry her, so I try to raise a hand to show her that I'm fine, b
 [sha:ed4e2df272653caab5fff593e6020ec60359d06c]{
 -- Page 770, Offset 12949.
 -- 呼吸―――呼吸はしていますね、心臓も動いている……なのに目を覚まさないなんて、ああ、もしかして頭が……！」
-He―――He's breathing. His heart is beating as well... But he isn't waking up, %{i}ahh%{/i}, maybe his head is...!"
+He―――He's breathing. His heart is beating as well... But he isn't waking up, ahh, maybe his head is...!"
```
Comment (rschlaikjer) [inline script/Ciel/Day 5/05_01_CIEL05_4C.txt:77]: Why dropped?

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 26]
```diff
@@ -179,21 +179,21 @@ I mean, even with a gag order, there's going to be leaks, right?"
 -- Page 798, Offset 13013.
 -- 「……アレでもちっと緊張感があれば使えるんだがな。あの捨て鉢さはなんだ、若い頃に挫折でもしたのかカリウス？」
 "...That guy would really come in handy if he had %{i}any%{/i} fucking sense of urgency at all.
- How'd he end up in such a sorry state...
+ How'd he end up in such a sorry state...?
```
Comment (rschlaikjer) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:182]: Implicit Q imo

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 27]
```diff
@@ -179,21 +179,21 @@ I mean, even with a gag order, there's going to be leaks, right?"
...
  His spirit get crushed when he was young or something, Karius?"
 }
 [sha:c99b4827f10eb2502f08740495726839c3e10818]{
 -- Page 799, Offset 13014.
 -- 「私にはとんと。安藤の私生活に興味はありませんので。
-"I haven't the slightest idea. Andou's private life holds no interest for me.
+"I haven't the slightest idea. Andou's private life holds no interest to me.
```
Comment (rschlaikjer) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:188]: Revert, `holds no interest for` is correct

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt (hunk 17 above)
Comment (rschlaikjer) [reply script/Ciel/Day 5/05_01_CIEL05_4D.txt:195]: Yeah this sentence is meant to be run-on. It's part of the joke.

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 28]
```diff
@@ -469,21 +469,21 @@ When you say 'opening act', which part was that exactly... sir?"
...
- You'd think that after being a disposable attack dog for ten years you'd have grown a fucking backbone.
+ You'd think that after being a disposable attack dog for ten years, you'd have grown a fucking backbone.
  The second I get back to the Vatican, I'm gonna have your head.
  And I don't mean I'll send you to a monastery―――I mean I'll send you to the fucking guillotine."
 }
 [sha:9d63530f81ce490fd69d486daec2efd03f2ac721]{
 -- Page 819, Offset 13065.
 -- 「へ―――？」
-"%{i}Huuh%{/i}―――?"
+"Huuh―――?"
```
Comment (rschlaikjer) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:486]: Revert

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 29]
```diff
@@ -769,7 +769,7 @@ I still can't quite believe it but... she really defeated an Ancestor, right?
 [sha:85bb9e1a2e68d5536f3fc9a41b30670b4fe91d0d]{
 -- Page 842, Offset 13113.
 -- 「はあ？　バカいってんじゃねえぞ<七面鳥|ターキー>。
-"%{i}Whaaa%{/i}? Where the fuck did you come up with that idea, you turkey.
+"Whaaa? Where the fuck did you come up with that idea, you turkey?
```
Comment (rschlaikjer) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:772]: My man what is the rhyme and reason for these exclamation changes

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt [hunk 30]
```diff
@@ -883,7 +883,7 @@ No Scriptures related to the Dead Apostles, no trace that vampire was ever there
 [sha:99882a66c8445515ef4650c199d99a4655d1846c]{
 -- Page 849, Offset 13134.
 -- 「いやあ、そんなワケのわからない<戦利品|モノ>押しつけられても困りますけどねぇ。
-"I meeannn, I wouldn't have a clue what to do with half of that stuff if it ended up being pawned off on me.
+"I meeannn, I wouldn't have a clue what to do with half of that stuff if it ended up being pawned off to me.
```
Comment (rschlaikjer) [inline script/Ciel/Day 5/05_01_CIEL05_4D.txt:886]: Revert, `pawn off on` correct

File: script/QA/QA_13.txt [hunk 31]
```diff
@@ -228,8 +228,9 @@ zone for long.
 [sha:284a0ca688b040e41aa5e624ada434461e4ba6fc]{
 -- Page 21, Offset 43355.
 -- 「いける！」と思った時こそ、
-It's times when you think you've got it all%{n}
-in the bag
+// text overflow, might need a bigger speech bubble
+It's the times when you think you've got it%{n}
```
Comment (rschlaikjer) [inline script/QA/QA_13.txt:232]: It's fine to elide `the` here

File: script/Ciel/Day 5/05_01_CIEL05_1B.txt (hunk 2 above)
Comment (Avuxo) [reply script/Ciel/Day 5/05_01_CIEL05_1B.txt:4]: yeah, i think revert

File: script/Ciel/Day 5/05_01_CIEL05_1E.txt (hunk 10 above)
Comment (Avuxo) [reply script/Ciel/Day 5/05_01_CIEL05_1E.txt:29]: to revert

File: script/Ciel/Day 5/05_01_CIEL05_1E.txt [hunk 32]
```diff
@@ -529,7 +529,7 @@ The woman I knew as my teacher has transformed into an outrageous sight before m
 [sha:9533d4fdd63cf6cfee57ceb605284418261798c5]{
 -- Page 238, Offset 11493.
 -- 「おとなしく潜入捜査やってれば調子に乗ってくれちゃって。
-"Aah, I can't believe I got all carried away with my timid little undercover act.
+"I can't believe I got all carried away with my timid little undercover act.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 5/05_01_CIEL05_1E.txt:532]: Yeah, just checked, not there /shrug

File: script/Ciel/Day 5/05_01_CIEL05_4D.txt (hunk 29 above)
Comment (rschlaikjer) [reply script/Ciel/Day 5/05_01_CIEL05_4D.txt]: Sorry, be exclamation here I was referring to the `Whaa?`. The ? after turkey is fine.

File: script/Ciel/Day 5/05_01_CIEL05_4C.txt (hunk 25 above)
Comment (rschlaikjer) [reply script/Ciel/Day 5/05_01_CIEL05_4C.txt:77]: It's definitely a blurry distinction. I guess this one could go either way, but there's some ineffable point where these go from being "ah, yes" where 'ah' is a word to "_Aaahh_! Yes!" where it's more of an exclamation and should maybe be italicized.

File: script/Ciel/Day 5/05_01_CIEL05_4.txt [hunk 33]
```diff
@@ -765,7 +765,7 @@ A thing that can never be understood.
 [sha:8a08e98afb8cc6416cf044eba6980e1b71c27c67]{
 -- Page 721, Offset 12791.
 -- 言語、記録で理論を構築する思考形態では見えないはずの、光を見た。
-A light that it shouldn't be possible to perceive with the same patterns of thought that build theories though language and documentation.
+A light that it shouldn't be possible to perceive with the same patterns of thought that build theories through language and documentation.
```
Comment (yk1512) [inline script/Ciel/Day 5/05_01_CIEL05_4.txt:768]: A light that shouldn't be able to be perceived

**Review by yk1512 — APPROVED** [review]
🙏

File: script/Ciel/Day 5/05_01_CIEL05_1E.txt (hunk 10 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 5/05_01_CIEL05_1E.txt:29]: but "It's" is correct right?

File: script/Ciel/Day 5/05_01_CIEL05_4C.txt (hunk 25 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 5/05_01_CIEL05_4C.txt:77]: Hmm I see what you mean now. Alright I'll add it back

**Pull request merged**